modules/transactionpool/accept.go

```diff
@@ -237,7 +237,18 @@ func (tp *TransactionPool) handleConflicts(ts []types.Transaction, conflicts []T
 		tp.knownObjects[ObjectID(diff.ID)] = setID
 	}
 	tp.transactionSetDiffs[setID] = cc
-	tp.transactionListSize += len(encoding.Marshal(superset))
+	tsetSize := len(encoding.Marshal(superset))
+	tp.transactionListSize += tsetSize
+
+	// debug logging
+	if build.DEBUG {
+		tp.log.Debugf("accepted transaction superset %v, size: %vB\n", setID, tsetSize)
+		for i, t := range superset {
+			tp.log.Debugf("superset transaction %v size: %vB\n", i, len(encoding.Marshal(t)))
+		}
+		tp.log.Debugf("tpool size is %vB after accepting transaction superset\n", tp.transactionListSize)
+	}
```

Comment:
Oh I forgot it was structured that way. Ok, looks good then. But does still need to be a single log call, even if multiple lines.